src/test/xml_coverage.go

```diff
@@ -48,16 +49,22 @@ func coverageResultToXML(sources []core.BuildLabel, coverage core.TestCoverage)
 	linesCovered := 0
 	validFiles := coverage.OrderedFiles()
 
+	cwd, err := os.Getwd()
```

Comment:
I don't think you should find the working directory here. Use `core.RepoRoot` if you need to know it - but I don't think you should be encoding absolute paths into the output file.

Reply:
I just tried it out, using relative path would show 0 covered in intellij. :(

Reply:
Intellij is terrible :(

I'm not sure what is best then. These files can get shared through the cache, having absolute paths in them is not a good idea.

Reply:
Perhaps you could write them with relative paths and have a separate post-process step that outputs the file for intellij that doesn't get cached (this is likely to help if we ever want to support other IDEs or formats too)?

Reply:
Hmmm on further thought the output file is *not* cached so I guess that's OK.

I still think it's rubbish that Intellij doesn't handle relative paths though.